subprojects/c-flags.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = c-flags-1.5.2
+source_url = https://github.com/DieTime/c-flags/archive/1.5.2.tar.gz
+source_filename = c-flags-1.5.2.tar.gz
+source_hash = f4f4aed5156dca60af2434cecfa45ccd706eef37bf8b731459ebd41ca1daf150
+
+[provide]
+cflags = cflags_dep
+cflags_static = cflags_static_dep
```

Comment:
This is interesting to see, as usually projects tend to rely on meson's `default_library={static|shared|both}` to generate whichever one is suitable. It's also something that a subproject fallback can set so that one links to the shared library if it's globally installed, but the static library if a subproject is used.

Reply:
Thanks, I changed the build script, now the library is built using the `library` command, so that the user can specify the type of library using `default_library`.

Reply:
Also fixed compatibility for Windows. The last PR fails on Windows workflow.